spec/features/variables/reverting_variable_spec.rb

```diff
@@ -78,14 +78,16 @@
             wait_for_ajax
           end
 
-          it 'reverts the variable to the correct revision and displays the correct revision information' do
+          it 'reverts the variable to the correct revision and displays the correct revision information and switches provider context' do
```

Comment:
sorry to be a pain, but since you're making this match the services one, I think it should have the test for the modal also...